airflow/cli/simple_table.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, *args, **kwargs):
         self.pad_edge = kwargs.get("pad_edge", False)
         self.box = kwargs.get("box", ASCII_DOUBLE_HEAD)
         self.show_header = kwargs.get("show_header", False)
+        self.header_style = kwargs.get("header_style", "bold green")
```

Comment:
I cannot relate this change to other changes in this PR, because there is no header explicitly specified for the `SimpleTable`s created in this module.

May you please help clarify a bit? Thanks.

Reply:
Otherwise all look good to me. Much much nicer & more comprehensible output now 👍

Reply:
Good catch, that should not be part of this change 👌